src/main/java/pl/nogacz/snake/application/AchievementHandler.java

```diff
@@ -0,0 +1,119 @@
+package pl.nogacz.snake.application;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class AchievementHandler {
+    
+    private final String pathToAchievements = "src/main/resources/AchievementData/Achievements.txt";
```

Comment:
Bu doğru bir path değil. Oyun bir jar dosyası olarak çalışacak. O zaman src/main gibi bir klasör de oyunu oynayanın bilgisayarında olmayacak. Bu pathi proje bağımsız bir path yap.

Reply:
Hocam bu sorunu engellemek için pathleri "AchievementData/Achievements.txt" şeklinde mevcut directorye yazacak (yoksa oluşturacak) şekilde yazmıştım fakat resourcesin içinde olmasının doğru olacağını söylediğiniz için değiştirmiştim. Program jar olarak kullanıldığında jarın içindeki resources'ta değişiklikler yapmak mümkün olmayacağı için bu kısmı tekrar eski haline getirmem doğru olur mu?